src/shared/components/TaskCard.jsx

```diff
@@ -1,35 +1,59 @@
 import styled from 'styled-components'
 import Avatar from './Avatar'
-import { IoMdHeart } from 'react-icons/io'
+import { IoIosArrowDown, IoIosArrowUp, IoMdHeart } from 'react-icons/io'
 import Text from './Text'
 import { themeColors } from 'shared/constants/colors'
 import useToggle from 'shared/hooks/useToggle'
 import Badge from './Badge'
+import { useRef } from 'react'
+import { useEffect } from 'react'
```

Comment:
```suggestion
import { useRef, useEffect } from 'react'
```
요거는 한번에 import 해도 될 것 같네용 ㅎㅎ